composer.json

```diff
@@ -50,7 +50,7 @@
   "require-dev": {
     "roave/security-advisories": "dev-master",
     "thetwopct/load-media-from-production": "dev-main",
-    "wp-coding-standards/wpcs": "^3.0.0",
+    "wp-coding-standards/wpcs": "^3.0.1",
```

Comment:
You don't need to make changes like this since the `^` rule allows non-breaking changes, that is, composer will grab the latest version up until 4.0.0 in this case. See [the description here](https://getcomposer.org/doc/articles/versions.md#caret-version-range-).

Reply:
I know but I wanted this to mirror our CNCF composer.json. On composer v2 there is a bump command to keep what's in composer.json in line with the composer-lock file. Makes seeing exactly where you are at with dependencies much easier and I'm used to using it. Except we're stuck on v1.